templates/single-product/add-to-cart/variable.php

```diff
@@ -15,11 +15,13 @@
 
```

Comment:
GitHub won't let me add a suggestion directly on line 11, but it should probably be changed to `@version WC-3.5.5` as was done in the corresponding WC commit: https://github.com/woocommerce/woocommerce/commit/7def966e4fe626cca274b5a714e07ac36f2655a6#diff-74ce9b72ea9f1b5b22481be8d8349d54R15

Reply:
New issue #231 created.

Reply:
Ideally this should be done in this same PR instead.

Reply:
Agree. #231 closed. I'll add a new commit tomorrow.

Reply:
Thanks for resolving this.